### PR TITLE
fix: fix for update and bulk array update operation  in postgres

### PR DESCRIPTION
**Pull request by saxenakshitiz**

### CODE REVIEW

**Review by suresh-prakash — COMMENTED** [review]
Looks good to me.

File: document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java [hunk 1]
```diff
@@ -570,5 +570,8 @@ public static void enrichPreparedStatementWithParams(
                 log.error("SQLException setting Param. key: {}, value: {}", k, v);
               }
             });
+    if (log.isDebugEnabled()) {
+      log.debug("Executing statement - preparedStatement: {}", preparedStatement);
```
Comment (kotharironak) [inline document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java:574]: `preparedStatement` has toString, right?

**Review by kotharironak — APPROVED** [review]
lgtm

File: document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java (hunk 1 above)
Comment (saxenakshitiz) [reply document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java:574]: Already checked this part. It works fine.

**Pull request merged**